src/audio/drc/drc_math_lib/asin/c_fixpt/drc_asin.c

```diff
@@ -0,0 +1,221 @@
+/*
+ * File: drc_asin.c
+ *
+ * Author :Shriram Shastry
+ * Copyright (c) 2016, Intel Corporation All rights reserved.
+ */
+
+/* Include Files */
+#include "drc_asin.h"
+
+/* Function Declarations */
+static int asr_s32(int u, unsigned int n);
```

Comment:
please try to order your functions so you don't need forward declarations for static functions.